ui/src/app/utils/workspacePermissions.ts

```diff
@@ -0,0 +1,43 @@
+import {Workspace, WorkspaceAccessLevel, WorkspaceResponse} from 'generated';
+
+/**
+ * Composite class with helper functions useful in the UI.
```

Comment:
Use your own judgement, but my gut says this information is obvious from the implementation. Nice helper class, though.

Reply:
Also actually Greg!